02/app.js

```diff
@@ -5,9 +5,14 @@ class Counter extends React.Component {
     state = {
         amount: 0,
     }
-    
+    handleClick = (e) => {
+        const { amount } = this.state;
+        if (e.target.classList.contains('btn')) {
```

Comment:
W React zazwyczaj nie musimy takich rzeczy sprawdzać (nie opieramy się o DOM), bo logikę mamy w komponencie i wiemy jaki to element. W przeciwieństwie do czystego JS-a, tutaj mamy również HTML pod ręką (patrz niżej JSX).